src/Uno.UI/Microsoft/UI/Xaml/Controls/Repeater/FlowLayoutState.cs

```diff
@@ -29,6 +29,8 @@ internal Size SpecialElementDesiredSize
 			set => m_specialElementDesiredSize = value;
 		}
 
+		internal double Uno_LastKnownAverageLineSize;
```

Comment:
Can it be tested? Maybe with multiple instances?

Reply:
Those patches are already tested by themselves, adding tests to validate that we are not moving them back to the `Layout` would be really difficult and would not prevent other state to be stored in the `Layout` ...